### ISSUE TITLE
Please extend README

### ISSUE THREAD

**Issue by guettli** [RedMoonStudios/hetzner#19]
Please extend README. Please tell me how to install this python library.

**Comment by mcat-ee** [comment]
@guettli I know this is an old ticket so you've likely moved on, but just in case further people have trouble with this - was the below command not working for you?

`pip3 install hetzner` (or maybe `sudo pip3 install hetzner` if you need sudo permissions)

**Comment by guettli** [comment]
Yes, I moved on. But why don't you add a "Install" section to the README?
